buildSrc/src/main/java/org/elasticsearch/gradle/internal/precommit/ForbiddenPatternsTask.java

```diff
@@ -65,7 +65,9 @@
         .exclude("**/*.jks")
         .exclude("**/*.crt")
         .exclude("**/*.keystore")
-        .exclude("**/*.png");
+        .exclude("**/*.png")
+        // vim swap file - included here to stop the build falling over if you happen to have a file open :-|
```

Comment:
I wonder if we should simply ignore hidden files (those with a `.` prefix) altogether?

Reply:
I can see the argument for that, but since no one is complaining about other file patterns at the moment, I'd prefer to stick with a conservative pattern.